lumicks/pylake/kymotracker/tests/test_refinement.py

```diff
@@ -54,3 +54,38 @@ def test_refinement_line(loc, inv_sigma=0.3):
     calibrated_image = CalibratedKymographChannel.from_array(np.expand_dims(image, 1))
     line = refine_lines_centroid([KymoLine([0], [25], image=calibrated_image)], 5)[0]
     np.testing.assert_allclose(line.coordinate_idx, loc, rtol=1e-2)
+
+
+def test_gaussian_refinement(kymo_2lines):
+    coordinates, expectation, photon_count, parameters = kymo_2lines
+    pixel_size = parameters[0].pixel_size
```

Comment:
Just out of curiosity, what is the pixel size in this test?

Reply:
100 nm, same as the experimental kymo data we've been testing on